test-framework/junit5-virtual-threads/src/main/java/io/quarkus/test/junit/virtual/internal/InternalEvents.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.test.junit.virtual.internal;
+
+import jdk.jfr.Category;
+import jdk.jfr.Event;
+import jdk.jfr.Label;
+import jdk.jfr.Name;
+import jdk.jfr.StackTrace;
+
+/**
+ * Internal events used during the capture.
+ */
+public interface InternalEvents {
+
+    String INITIALIZATION_EVENT_NAME = "io.quarkus.test.junit.virtual.internal.InternalEvents.InitializationEvent";
+    String SHUTDOWN_EVENT_NAME = "io.quarkus.test.junit.virtual.internal.InternalEvents.ShutdownEvent";
+
+    String CAPTURING_STARTED_EVENT_NAME = "io.quarkus.test.junit.virtual.internal.InternalEvents.CapturingStartedEvent";
+    String CAPTURING_STOPPED_EVENT_NAME = "io.quarkus.test.junit.virtual.internal.InternalEvents.CapturingStoppedEvent";
+
+    @Name(INITIALIZATION_EVENT_NAME)
+    @Category("loom-unit")
```

Comment:
Shall we keep the category name? I am not sure if it is relevant.

Reply:
Good point, I fixed it!